training_program/serializers.py

```diff
@@ -1,27 +1,13 @@
 '''Define how to serialize our models.'''
 from rest_framework import serializers
 
-from training_program.models import (
-    Program, ProgramCategory, ProgramForm
-)
-
-
-class ProgramCategorySerializer(serializers.ModelSerializer):
-    '''Indicate how to serialize ProgramCategory instance.'''
-    class Meta:
-        model = ProgramCategory
-        fields = '__all__'
-
-
-class ProgramFormSerializer(serializers.ModelSerializer):
-    '''Indicate how to serialize ProgramForm instance.'''
-    class Meta:
-        model = ProgramForm
-        fields = '__all__'
+from training_program.models import Program
 
 
 class ProgramSerializer(serializers.ModelSerializer):
     '''Indicate how to serialize Program instance.'''
+    status_str = serializers.CharField(source='get_category_display',
```

Comment:
Why is `status_str`, this field has nothing to do with `status`. **PLEASE** understand what you're doing before coding, **DO NOT** copy and paste.